rllib/tuned_examples/dqn/stateless-cartpole-r2d2-fake-gpus.yaml

```diff
@@ -0,0 +1,29 @@
+stateless-cartpole-r2d2:
+    env: ray.rllib.examples.env.stateless_cartpole.StatelessCartPole
+    run: R2D2
+    stop:
+        episode_reward_mean: 150
+        timesteps_total: 1000000
+    config:
+        # Works for both torch and tf.
+        framework: tf
+        num_workers: 0
```

Comment:
just to double check, should we add a non-local worker here?

Reply:
You are probably right. I think I tried to reduce the number of CPUs this needs on the (expensive) GPU machine, but this is fake-GPUs, so it's fine. Will change.